release/mk-release-snapshots.sh

```diff
@@ -9,23 +9,40 @@
 SQUID_RELEASE=`ls -1 $SQUID_VCS_PATH | cut -d- -f2 | sort -h | tail -n 1`
 
 # Hostname of the CI server where snapshots are built
-SQUID_BUILD_SERVER=buildmaster.squid-cache.org
+SQUID_BUILD_SERVER=build.squid-cache.org
 
 # file Jenkins uses to assemble the list of snapshot files
 outfile=HEAD.out
 
+# arguments:
+# jenkins job name
+# destination directory
+get_artifacts() {
+	local job=$1
+	local outdir=$2
+	local artifactsUrl="https://$SQUID_BUILD_SERVER/job/${job}/lastSuccessfulBuild/artifact/artifacts/*zip*/artifacts.zip"
+	wget --quiet "$artifactsUrl"
+	if [ ! -e artifacts.zip ] ; then
+		echo "could not download artifacts from $artifactsUrl"
+		return 1
+	fi
+	unzip -qq artifacts.zip
+	if [ ! -d artifacts ]; then
+		echo "${job}:artifacts.zip does not contain a subdirectory"
+		return 1;
```

Comment:
Please cleanup all files created on exit.
```suggestion
		rm -d `unzip -Z1 artifacts.zip`
		rm -f artifacts.zip
		return 1
```

Reply:
good idea